src/Pages/Order/Order.js

```diff
@@ -1,84 +1,219 @@
+        phone_number: this.state.user.phone_number,
+        email: this.state.user.email,
+        delivery_address: this.state.fullAddress,
+        postal_code: this.state.zoneCode,
+        detailed_address: this.state.detailed_address,
+        customor_message: this.state.customor_message,
+      },
+      user: {
+        add_my_address: this.state.add_my_address,
+        point_used: Number(this.state.point_used),
+        point:
+          this.state.cartItems?.reduce((acc, cur) => acc + cur.total_price, 0) *
+          0.01,
       },
     });
-  };
+    const totalPrice = this.state.cartItems?.reduce(
+      (acc, cur) => acc + cur.total_price,
+      0
+    );
+    fetch('http://10.58.2.56:8000/order', {
+      method: 'POST',
+      headers: {
+        Authorization:
+          'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VyX2lkIjoyfQ.wlCljldMPYhX12CrF2N1-nCSvDqf_HXKYFd68gFQPVY',
+        //sessionStorage.getItem('token'),
+      },
+      body: JSON.stringify({
+        products: orderProduct,
+        receiver: {
+          name: this.state.user.name,
+          phone_number: this.state.user.phone_number,
+          email: this.state.user.email,
+          delivery_address: this.state.fullAddress,
+          postal_code: this.state.zoneCode,
+          detailed_address: this.state.detailed_address,
+          customor_message: this.state.customor_message,
+        },
+        user: {
+          add_my_address: this.state.add_my_address,
+          point_used: Number(this.state.point_used),
+          point: 3450,
+        },
 
-  componentDidMount() {
-    fetch('http://localhost:3000/data/cartItems.json')
+        //
+      }),
+    }) //
       .then(res => res.json())
       .then(res => {
         console.log(res);
-        this.setState({
-          cartItems: res.cartItems,
-        });
+        // if (response.status === 400) {
+        //   alert('다시 한번 확인해주세요');
+        // } else {
+        alert('구매완료!');
+        this.props.history.push('/main');
+        //   window.location.reload();
       });
-  }
+  };
+
+  //주소 변경 체크박스
+  handleAddMyAddress = () => {
+    if (this.state.add_my_address) {
+      this.setState({
+        add_my_address: 0,
+      });
+    } else {
+      this.setState({
+        add_my_address: 1,
+      });
+    }
+  };
+
+  //주소 API 연결 및 등록
+  handleOpenPost = () => {
+    this.setState({
+      isDaumPost: true,
+    });
+  };
+
+  handleAddress = data => {
+    let AllAddress = data.address;
+    let extraAddress = '';
+    let zoneCodes = data.zonecode;
+
+    if (data.addressType === 'R') {
+      if (data.bname !== '') {
+        extraAddress += data.bname;
+      }
+      if (data.buildingName !== '') {
+        extraAddress +=
+          extraAddress !== '' ? `, ${data.buildingName}` : data.buildingName;
+      }
+      AllAddress += extraAddress !== '' ? ` (${extraAddress})` : '';
+    }
+    this.setState({
+      fullAddress: AllAddress,
+      zoneCode: zoneCodes,
```

Comment:
key, value값이 같을 때에는 한번만 작성해주시면 됩니다. 
```suggestion
      zoneCode,
```

Reply:
네 알겠습니다. 그런데 저기는 키 벨류가 다릅니다!